index.html

```diff
@@ -8,60 +8,60 @@
     <title>Miguel Cuello's Portfolio</title>
 </head>
 <body>
+
     <header>
       <nav>
         <a class="navbar-logo" href="#">
             My Logo
         </a>
-
         <label class="navbar-hamburger" for="toggle">
             <span class="bar"></span>
             <span class="bar"></span>
             <span class="bar"></span>
         </label>
-
-
-        <ul class="nav-list">
-            <li class="nav-item">
-                <a class="nav-link" href="#portfolio">Portfolio</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#about">About</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#contact">Contact</a>
-            </li>
-        </ul>
       </nav>
     </header>
+
+
+    <section class="container">
+
         <div id="ellipse1"></div>
-        <div id="rectangle"></div>
+        <div id="rentangle"></div>
         <div id="ellipse5"></div>
         <div id="rectangle2"></div>
         <div id="ellipse3"></div>
-        <div id="circle"></div>
+        <div id="circle1"></div>
         <div id="ellipse2"></div>
         <div id="ellipse4"></div>
         <div id="circle2"></div>
         <div id="circle3"></div>
         <div id="rectangle3"></div>
 
-    <section class="about-me">
-        <h1 class="primary-text">
+    <h1 class="primary-text">
             I’m Miguel Cuello.
 Glad to see you!
         </h1>
         <p class="first-text">
             I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
         </p>
         <h3>LET’S CONNECT</h3>
-        <div id="social">
-            <img src="/images/twitter.svg" alt="twitter_logo" id="twitter" class="icon">
-            <img src="/images/in.png" alt="linkedin_logo" id="linkedin" class="icon">
-            <img src="/images/M.svg" alt="capital_m" id="m" class="icon">
-            <img src="/images/git.svg" alt="github_logo" id="github" class="icon">
-            <img src="/images/hand.svg" alt="hand" id="hand" class="icon">
-        </div>
+
+    <div class="social">
+
+      <div id="twitter"><a href="#twitter"></a>
+      </div>
+      <div id="linkedin"><a href="#likedin"></a>
+      </div>
+      <div id="m"><a href="#m"></a>
+      </div>
+      <div id="github"><a href="#git"></a>
+      </div>
+      <div id="hand"><a href="#hand"></a>
+      </div>
+
+</div>
```

Comment:
Please understand this that If an image/icon is NOT part of the content (aesthetic reasons) use CSS background-image like for example backgrounds that we setup we don't need to interact with BUT if the image/icon is part of the content like for example social media icons which we are using so if user clicks on them they can we our social media profiles an other example of images that are part of content are profile pictures.  Hope this clears it. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

- [ ] I would suggest not using the social media icons as a background image they need to be used inside an `img` tag. Also please use list tags (`ul` and `li`)to in-list the social media icons and inclose each icon into a hyperlink tag (`<a>`) . Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Example snippet:
 
```html

<ul id="social">
            <li><a href="#"><img src="/images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
            <li><a href="#"><img src="/images/in.png" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
            <li><a href="#"><img src="/images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
            <li><a href="#"><img src="/images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
            <li><a href="#"><img src="/images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
        </ul>
```

Reply:
I like this review so much, thanks you Ayesha. I'm not going to lie, I didn't know that we could to ask for review request over here, I've sent it before your advice :+1: 
It was to useful to me :100: